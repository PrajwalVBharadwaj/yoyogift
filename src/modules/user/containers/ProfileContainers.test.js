import React from "react";
import { shallow, cardDetails } from "../../../Enzyme";
import { ProfileContainer } from "./ProfileContainers";

describe("Testong ProfileContainer", () => {
  it("Snapshot testing", () => {
    const wrapper = shallow(<ProfileContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
