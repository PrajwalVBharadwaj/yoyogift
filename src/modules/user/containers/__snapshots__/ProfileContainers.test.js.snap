// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testong ProfileContainer Snapshot testing 1`] = `ShallowWrapper {}`;
